ci/cloudbuild/build.sh

```diff
@@ -306,9 +306,7 @@ if [[ "${DOCKER_FLAG}" = "true" ]]; then
     "--build-arg=NCPU=$(nproc)"
     -f "ci/cloudbuild/dockerfiles/${DISTRO_FLAG}.Dockerfile"
   )
-  if [[ "${TRIGGER_TYPE}" == "manual" ]]; then
-    build_flags+=("--quiet")
-  fi
+  export DOCKER_BUILDKIT=1
```

Comment:
oh noes, my precious vertical space...

Reply:
This output is very limited. But I suspect you just need to increase your history size (or whatever the setting is) in your terminal to something like 50k.

Reply:
The Kokoro builds seemed stuck, I restarted them.